portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Question data type.
+ */
+function Question(question, options, correct) {
+  this.question = question;
+  this.options = options;
+  this.correct = correct; // index where correct answer is
+}
+function presentQuiz() {
+  // create array of quiz questions 
+  let questions;
+
+  let quizContainer = document.getElementById('quiz-container');
+  for (q in questions) {
+    // display the question in a container
+    quizContainer.innerText = q.question;
+    // Display the options
+    // Make another container
+    // Use radio buttons
+    // Have a submit button
+    // When user clicks submit button display correct answer
```

Comment:
The `innerText` of `quizContainer` is getting set - but then it sounds like we'll need to add more containers in side of it? Does it make sense to just handle all of the questions in the same fashion?

Reply:
Indeed, more containers will be needed inside `quizContainer`, or in addition to `quizContainer`. I plan to show one question at a time, and show them in the same fashion. Hence why I'm handling them in the same fashion.

Reply:
I would encode as much structure as you can in the HTML source - so I think if you modify the HTML source to have a general structure like:
questionAndAnswerDiv
- questionDiv
- answersDiv
- submitDiv

and drop the line here about making another container then this makes sense. WDYT?